assets/patterns/header/README.md

```diff
@@ -0,0 +1,56 @@
+# Header
+
+{{ wip(4) }}
+
+This is the [GOV.UK header](https://www.gov.uk/service-manual/design/add-the-govuk-header-and-footer) with a number of additional parts, depending on the needs of your users.
+
+## When to use this pattern
+
+Start with the [GOV.UK header](https://www.gov.uk/service-manual/design/add-the-govuk-header-and-footer). All elements that are imported from the GOV.UK header must not be changed.
+
+Only add an HMRC header if there is a user need for people to know they are dealing with HMRC. For example, if they are making a payment and need to be sure that the money is going to the right government department.
+
+{{ example('header--hmrc-logo.html', true) }}
+
+If your service is in the personal tax account, use the [personal tax account header](/components/account-header/index.html) instead.
```

Comment:
Does this need updating in line with the most recent draft for the account header?

> If your service is part of an account, use the account header instead.

Reply:
I think Michael is updating this on SDT-788

Reply:
Ok cool. I'll add a note to that ticket to update this paragraph too (cc @mikeash82).